WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -42,7 +42,12 @@ - (BOOL)fb_isVisible
   if ([FBConfiguration shouldUseTestManagerForVisibilityDetection]) {
     return [(NSNumber *)[self fb_attributeValue:FB_XCAXAIsVisibleAttribute] boolValue];
   }
-  CGRect appFrame = self.application.frame;
+  CGRect appFrame;
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"11.0")) {
```

Comment:
This implementation works locally with Xcode 9 running iOS 11 and 10.3. And Xcode 8.3 running iOS 10.3.